libmuscle/cpp/build/msgpack/Makefile

```diff
@@ -29,7 +29,7 @@ $(MSGPACK_SRC): msgpack-$(msgpack_VERSION).tar.gz
 $(MSGPACK_LIB): $(MSGPACK_SRC)
 	cd msgpack-$(msgpack_VERSION) && rm -rf build && mkdir -p build && cd build && \
 		export PKG_CONFIG_PATH=$(PKG_CONFIG_PATH):$(PKG_CONFIG_EXTRA_DIRS) && \
-		cmake -DCMAKE_INSTALL_PREFIX=$(CURDIR)/msgpack -DMSGPACK_CXX11=ON -DMSGPACK_BUILD_EXAMPLES=OFF -DMSGPACK_BUILD_TESTS=OFF .. && \
+		cmake -DCMAKE_POLICY_VERSION_MINIMUM=3.5 -DCMAKE_INSTALL_PREFIX=$(CURDIR)/msgpack -DMSGPACK_CXX11=ON -DMSGPACK_BUILD_EXAMPLES=OFF -DMSGPACK_BUILD_TESTS=OFF .. && \
```

Comment:
This seems to be unrelated to the rest, and should be in a separate PR. I know what issue it solves and this is the right solution, just not here.